### PR TITLE
Updated to use Prisma 1.9

### PR DESCRIPTION
**Pull request by christiannwamba**

### CODE REVIEW

**Comment by christiannwamba** [comment]
@nicholaspretorius I just updated https://github.com/howtographql/howtographql/pull/672/commits/367ec70cb7d9dcad46d63bc68c44b44526c4e25b

File: content/backend/graphql-js/5-connecting-server-and-database.md [hunk 1]
```diff
@@ -272,6 +270,23 @@ The Prisma CLI also uses information that's provided in `.graphqlconfig.yml`. Th
...
+Update `prisma.yml` to include a deploy hook:
+
+```bash(path=".../hackernews-node/database/prisma.yml")
+endpoint: `https://eu1.prisma.sh/public-graytracker-771/hackernews-node/dev
+datamodel: datamodel.graphql
+
+# Deploy hook
+hooks:
+  post-deploy:
+    - graphql get-schema --project database
```
Comment (nikolasburk) [inline content/backend/graphql-js/5-connecting-server-and-database.md]: I think we're missing instructions in the tutorial to install the GraphQL CLI, so as of now the invokation of the hook is likely to fail if the reader doesn't have the CLI installed globally

File: content/backend/graphql-js/5-connecting-server-and-database.md [hunk 2]
```diff
@@ -272,6 +270,23 @@ The Prisma CLI also uses information that's provided in `.graphqlconfig.yml`. Th
...
+```
+</Instruction>
+
+A deploy hook is invoked when Prisma is done with deploying. In this case, we want to download the schema using the `get-schema` command and pointing to the `database` project that was configured in `.graphqlconfig.yml`.
```
Comment (nikolasburk) [inline content/backend/graphql-js/5-connecting-server-and-database.md]: I really like this explanation! The only things I'd slightly adjust is to make clear that the `graphql get-schema` command is based on the [GraphQL CLI](https://oss.prisma.io/content/graphql-cli/01-Overview.html) and probably add an instruction to install the CLI globally.

File: content/backend/graphql-js/5-connecting-server-and-database.md [hunk 3]
```diff
@@ -205,7 +204,6 @@ So, here's the trick. You're instantiating `Prisma` with the following pieces of
 
 - `typeDefs`: This points to the Prisma database schema which defines the full CRUD GraphQL API of Prisma. Note that you actually don't have this file yet - we'll tell you in a bit how to get it.
 - `endpoint`: This is the endpoint of your Prisma API. Don't forget to replace it with the endpoint of your own Prisma service here!
-- `secret`: Recall that all requests against the Prisma API need to be authenticated by including a JWT in the `Authorization` header of the HTTP request? This JWT needs to be signed with the `secret` defined in `prisma.yml`. As you're not making any _direct_ requests against the Prisma API, but these requests are being made for you by the `Prisma` binding instance, you need to tell it what that secret is so it can generate a JWT which it attaches to the requests.
```
Comment (nikolasburk) [inline content/backend/graphql-js/5-connecting-server-and-database.md]: As I mentioned in the other review, I'd probably keep the `secret` in `prisma.yml` as it's very much best practice.

File: content/backend/graphql-js/5-connecting-server-and-database.md [hunk 4]
```diff
@@ -192,7 +192,6 @@ const server = new GraphQLServer({
     db: new Prisma({
       typeDefs: 'src/generated/prisma.graphql',
       endpoint: 'https://eu1.prisma.sh/public-graytracker-771/hackernews-node/dev',
-      secret: 'mysecret123',
```
Comment (nikolasburk) [inline content/backend/graphql-js/5-connecting-server-and-database.md]: Again, same here - if we have the `secret` in `prisma.yml`, we need it in the constructor.

**Pull request merged**